index.html

```diff
@@ -0,0 +1,133 @@
+              </li>
+              <li class="feature-item">
+                <b>Только ночь, только хардкор</b>
+                <p>Работать днем, как все? Мы против этого. Ближе к полуночи работа только начинается.</p>
+                <a class="button" href="#">Узнать больше</a>
+              </li>
+            </ul>
+        </section>
+        <section class="services">
+            <h2 class="visually-hidden">Услуги</h2>
+            <ul class="services-list">
+                <li class="services-item">
+                    <h3>ВЕБ-САЙТЫ</h3>
+                    <p>Мир никогда не будет прежним после того как увидит ваш сайт</p>
+                    <a class="button" href="#">Заказать</a>            
+                </li>
+                <li class="services-item">
+                    <h3>ПРИЛОЖЕНИЯ</h3>
+                    <p>Покорите топ-10 приложений в AppStore и Google Play</p>
+                    <a class="button" href="#">Заказать</a>            
+                </li>
+                <li class="services-item">
+                    <h3>ПРЕЗЕНТАЦИИ</h3>
+                    <p>Вы даже не подозреваете, насколько вы изумительны</p>
+                    <a class="button" href="#">Заказать</a>            
+                </li>
+              
+            </ul>
+          </section>
+          <section class="aboutus">
+              <b>Мы - маленькая, но гордая дизайн-студия из Краснодара.</b>
+              <p>Исповедуем принципы минимализма и чистоты. Ставим математический расчет превыше креатива. Работаем не покладая рук, как роботы</p>
+              <h3>ВПОЛНЯЕМ ЗАКАЗЫ НА РАЗРАБОТКУ:</h3>
+              <ul>
+                  <li>Веб-сайтов любой сложности</li>
+                  <li>Мобильных приложений для IOS и Android</li>
+                  <li>Слайдшоу и видео для корпоративных презентаций</li>                 
+                                 
+              </ul>
+              <h3>С 2004 года любим точность во всем:</h3>
+              <ul>
+                  <li>
+                      <p>146%<br>
+                        Уровень самоотдачи
+                      </p>
+                  </li>
+                  <li>
+                    <p>100%<br>
+                      Соблюдение сроков
+                    </p>
+                  </li>
+                  <li>
+                    <p>50%<br>
+                      Минимальная предоплата
+                    </p>
+                  </li>                
+              </ul>
+          </section>
+          <section class="jobs">
+              <ul class="jobs-list">
+                <li>
+                    <a href="https://htmlacademy.ru/intensive/htmlcss">
+                        <img src="img/htmlacademy.jpg" width="220" height="165" alt="HTML academy">                    
+                    </a>
+                </li>     
+                <li>
+                    <a href="https://http://barbershop-borodinskiru/">
```

Comment:
```suggestion
                    <a href="https://barbershop-borodinski.ru/">
```
это в общем не важно, но чтоб ссылка имела правильный вид